packages/inform/src/Adaptor.js

```diff
@@ -1,6 +1,5 @@
 import { expandReferences, encode } from '@openfn/language-common/util';
 import * as util from './Utils';
-import { fetch } from 'undici';
```

Comment:
@hunterachieng looks like `fetch` is need in `downloadAttachment()` function. See https://github.com/OpenFn/adaptors/blob/main/packages/inform/src/Adaptor.js#L210

Reply:
@mtuchi  Yes. But to use fetch, I do not have to import from `undici` its redundant since it exists within node automatically. But `undici` is being used in tests.

Reply:
Yep, `fetch()` is native so it's fine not use the undici import. You can probably remove the dev dependency from the package too.

Reply:
Ah you're right - OK, fair enough!